lib/ansible/modules/network/meraki/meraki_organization.py

```diff
@@ -25,6 +25,7 @@
         description:
         - Create or modify an organization.
         - C(org_id) must be specified if multiple organizations of the same name exist.
+        - C(absent) WILL DELETE YOUR ENTIRE ORGANIZATION WITHOUT CONFIRMATION. USE WITH CARE.
```

Comment:
If this also deletes associated objects to your organization, maybe also mention that. Make it scary enough so that people really only use it when they're sure (and can't claim "the documentation didn't say that" afterwards if they still did).